Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -26,11 +26,12 @@
 
 WI.InlineSwatch = class InlineSwatch extends WI.Object
 {
-    constructor(type, value, readOnly)
+    constructor(type, value, {readOnly, preventChangingColorFormats} = {})
     {
         super();
 
         this._type = type;
+        this._preventChangingColorFormats = preventChangingColorFormats;
```

Comment:
since this is optional we should add a `!!` to coerce it to a boolean (ditto below for `readOnly`)

NIT: Should we `console.assert` that this is only provided when `WI.InlineSwatch.Type.Color`?